src/components/Session/SignOut.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withFirebase } from '../Firebase';
+
+const SignOutButton = ({ firebase }) => (
+  <button type='button' id='btn_logout' onClick={firebase.doSignOut}>
```

Comment:
Handling the button clicks are more related to issue #17

Unit tests are missing